clang/test/3C/type_params_macro.c

```diff
@@ -18,15 +18,15 @@ void test_none() {
   int *i = 0;
   foo(i);
 }
-// CHECK: void test_none() _Checked {
+// CHECK: void test_none() {
```

Comment:
Should this not have been checked? If not, why wouldn't have the compilation by `clang`, post conversion, discovered the issue? Perhaps the test was not doing that part?

Reply:
I guess this is a general question: It would be nice to confirm that the added annotation makes the Checked C compiler happy, so we should make sure that these tests are doing that, for `-addcr` runs.

Reply:
This happens because of the regression I mentioned above. It's fine to call a function who's parameters are unchecked but have itypes in a checked region, but the change I made to fix one issue revealed this one. If it was an issue that caused miscompilation we would have caught it, and in fact the tests did catch such an issue. Working on fixing this now but I though you might want this change in sooner because of the example code.

Reply:
Fixed in latest commit